### ISSUE TITLE
Add any single line python programs

### ISSUE THREAD

**Issue by girishtulabandu** [girishtulabandu/One_line_PythonPrograms#2]
* You can contribute any python programs that can be written in a single line. 
* Kindly read the readme file before making a pull request.
* Follow the contribution guidelines properly.
* Add a comment under this issue once you raise the PR with the number.

Happy coding. All the best 👍 

Edit1: For problem-solving and brainstorming, consider solving this [#1](https://github.com/girishtulabandu/guvi_tech/issues/1) from my other repository.

Edit2: Most of the people are not including sample input and expected output as mentioned in the [contribution guidelines.](https://github.com/girishtulabandu/One_line_PythonPrograms#contribution-guidelines). We can not merge the codes that are against to our contribution guidelines.

**Comment by vivekkj123** [comment]
kindly assign me

**Comment by girishtulabandu** [comment]
> kindly assign me

This is a general issue, anyone can contribute. Just go ahead and raise the pr

**Comment by MalwarePerseus** [comment]
Hi! can we submit Multiple One Liners?
I've Made a PR, please check.

**Comment by girishtulabandu** [comment]
> Hi! can we submit Multiple One Liners?
> I've Made a PR, please check.

Yeah but please use a separate file for each program(one-liner). Will check the pr shortly

**Comment by abhishektandon-github** [comment]
I have made a pull request #15 
kindly review and merge

**Comment by abhishektandon-github** [comment]
kindly review #20 and merge

**Comment by Ajay0511** [comment]
I would like to work on this.

**Comment by gurkiratsingh-1** [comment]
kindly review #34  and merge

**Comment by cryp73r** [comment]
Submitted Number Tree Printing... #31

**Comment by abhishektandon-github** [comment]
kindly review #37 and merge

**Comment by abhinav-193** [comment]
Kindly reiview #42 and merge.

**Comment by Maitreyi88** [comment]
kindly review #47   and merge

**Comment by omkumar40** [comment]
> * You can contribute any python programs that can be written in a single line.
> * Kindly read the readme file before making a pull request.
> * Follow the contribution guidelines properly.
> * Add a comment under this issue once you raise the PR with the number.
> 
> Happy coding. All the best 👍

#50

**Comment by Ajay0511** [comment]
> * You can contribute any python programs that can be written in a single line.
> * Kindly read the readme file before making a pull request.
> * Follow the contribution guidelines properly.
> * Add a comment under this issue once you raise the PR with the number.
> 
> Happy coding. All the best 👍

Please review #44

**Comment by nutanaarohi123** [comment]
would like to work on this issue.

**Comment by Wolfy7** [comment]
kindly review #53 and merge

**Comment by OfficialNMN** [comment]
Hi, I have created MergeSort.py file please review my pull request OfficialNMN:patch-1

**Comment by SkaneroOo** [comment]
Please check #56

**Comment by sreyareddy15** [comment]
PR #57  
kindly merge

**Comment by girishtulabandu** [comment]
I request you all whose PRs have not merged yet, are not following[ contribution guidelines](https://github.com/girishtulabandu/One_line_PythonPrograms#contribution-guidelines). Kindly update as per the contribution guidelines. Otherwise, we'll be marking it as "INVALID"

**Comment by omkumar40** [comment]
Please review #50 added the changes requested

**Comment by abhishektandon-github** [comment]
@girishtulabandu add `hacktoberfest2020` and `hacktoberfest` to your repository please, acc to the new guidelines

**Comment by gurkiratsingh-1** [comment]
Sir, I've pr 6 days ago and not got merged,kindly review #34 and merge

**Comment by nutanaarohi123** [comment]
Kindly review @girishtulabandu  PR #96

**Comment by omkumar40** [comment]
Kindly review #50

**Comment by isabellabond** [comment]
Hello, please review #90 @girishtulabandu 
Thank you!

**Comment by poison-pizza** [comment]
Hi, please review #105

**Comment by prathapnsvr** [comment]
added a one-liner for finding the maximum sum subarray given a list of positive or negative numbers. please review #117 @girishtulabandu
